FluentUI/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,5 @@
   ~ Copyright (c) Microsoft Corporation. All rights reserved.
   ~ Licensed under the MIT License.
   -->
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.microsoft.fluentui" />
+    package="com.microsoft.fluentui_container" />
```

Comment:
Is this necessary? container name?
My only concern is package name here says container and in gradle we just say fluentui

Reply:
It can be changed. But, in fluentui_others we have kept package name as com.microsoft.fluentui.
There, its kept as is because fluentui_others represents the controls which are not yet modularized. Hence no code has been changed there. 
Import statement in all files in it still looks like, import com.microsoft.fluentui.R

If we change package name in manifest, this import will have to be changed in all files of fluntui_others.

So, to keep pkg names unique, and as fluentui is used for others, gave this one a different name

Reply:
Just add a TODO to change it to fleuntui after full modularization.

Reply:
> you can add a TODO to change it to fleuntui after full modularization.

Done